Tests/MessagingInApp/Views/InAppMessageViewTest.swift

```diff
@@ -188,31 +183,35 @@ class InAppMessageViewTest: UnitTest {
 
         let inlineView = InAppMessageView(elementId: givenInlineMessage.elementId!)
 
-        await onDoneRenderingInAppMessage(givenInlineMessage)
+        await onDoneRenderingInAppMessage(givenInlineMessage, insideOfInlineView: inlineView)
 
-        XCTAssertTrue(isDisplayingInAppMessage(inlineView))
+        XCTAssertTrue(isInlineViewVisible(inlineView))
 
         // Simulate message expiration.
         await simulateSdkFetchedMessages([])
 
-        XCTAssertFalse(isDisplayingInAppMessage(inlineView))
+        XCTAssertFalse(isInlineViewVisible(inlineView))
+        XCTAssertNil(getInAppMessage(forView: inlineView))
     }
 
-    // Once an in-app message has been displayed it will not be replaced with another message.
-    // We plan to change this behavior in the future. Test function can be modified to match the new behavior at that time.
```

Comment:
This PR is the first PR that deals with replacing 1 in-app message with another one. Previously, once a View was showing a message we would not replace it's contents at all. 

There are some test functions, including this one, that were created matching the original behavior. Now that we are replacing in-app messages, these tests needed modified to show new expected behavior of replacing in-app messages.